src/python/cli_new/lib/cli/mesos.py

```diff
@@ -68,6 +63,35 @@ def get_agents(master, config):
     """
     endpoint = "slaves"
     key = "slaves"
+    return get_key_endpoint(key, endpoint, master, config)
+
+def get_framework_address(framework_id, master, config):
+    """
+    Given a master and an framework id, return the framework address
+    by checking the /master/frameworks endpoint of the master.
+    """
+    frameworks = get_frameworks(master, config)
+
+    for framework in frameworks:
+        if framework["id"] == framework_id:
+            return framework["webui_url"]
+    # pylint: disable=line-too-long
+    raise CLIException("Unable to find framework '{id}'".format(id=framework_id))
```

Comment:
Maybe better to indent on a new line than disable the pylint warning?

Reply:
:joy:don't know why I even add it.